themeconfigurator.php

```diff
@@ -498,6 +500,13 @@ public function getContent()
 			Configuration::updateValue('PS_TC_ACTIVE', (int)Tools::getValue('live_conf'));
 			Configuration::updateValue('PS_GRID_PRODUCT', (int)Tools::getValue('grid_list'));
 			Configuration::updateValue('PS_SET_DISPLAY_SUBCATEGORIES', (int)Tools::getValue('sub_cat'));
+			Configuration::updateValue('PS_TC_CUSTOM_CSS', Tools::getValue('PS_TC_CUSTOM_CSS'), true);
```

Comment:
Is it OK to call `Configuration::updateValue()` even if `PS_TC_CUSTOM_CSS` does not exist ?

I'm not familiar with configuration management 😅 shall we do something like
```
if (Configuration::getValue(PS_TC_CUSTOM_CSS')) {
    Configuration::createValue('PS_TC_CUSTOM_CSS');
}
```

Reply:
@matks yes it's ok, there's no such a thing as "create value", if configuration key doesn't exists it would be created